### DISCUSSION TITLE
Damping matrix for ross.ShaftElement and ross.ShaftElement6DoF

### DISCUSSION THREAD

**Discussion by ghost** [petrobras/ross#967] [Q&A] [answered]
Hi, thank you very much for this Python package, it is very useful. However, I have a question about the damping matrix in 'ross.ShaftElement' and 'ross.ShaftElement6DoF'. Why is there no consideration of internal damping in the shaft in 'ross.ShaftElement' (C = np.zeros((8, 8))) while 'ross.ShaftElement6DoF' considers internal damping in the shaft (C = self.alpha * self.M() + self.beta * self.K())?

**Comment by raphaeltimbo** [comment, marked as answer]
Hi @TeinIHuang!
There is actually no good reason for that...

The 4 dof has been implemented first, and then we had different people implementing the 6 dof and they decided to consider the internal damping.

In the future, we wish to solve this by making the 6 dof element the ‘default’, but for know we have this difference between them because of a lack of time for developers to standardize that.
